components/layout/NavigationLogo.tsx

```diff
@@ -4,14 +4,15 @@ import React from 'react';
 import ImageContainer from './ImageContainer.tsx';
 
 interface NavigationLogoProps {
+  sx?: object;
   fullSize?: boolean;
 }
 
-export default function NavigationLogo({ fullSize = false }: NavigationLogoProps) {
-  const imgSrc = fullSize ? '/cvlogo-sm.svg' : '/cvlogo-lg.svg';
-  const imgWidth = fullSize ? 50 : 275;
+export default function NavigationLogo({ sx, fullSize = false }: NavigationLogoProps) {
+  const imgSrc = '/cv-logo1.svg';
```

Comment:
Let's make sure we clean up all of the logos in the public folder and adjust the names so that we just use "logo" and not "logo1"

Reply:
Fixed